user_docs/en/changes.t2t

```diff
@@ -8,11 +8,11 @@ There is a new feature called sound split.
 This allows splitting NVDA sounds into one channel (e.g. left) while sounds from all other applications are placed in the other channel (e.g. right).
 
 There are new commands for modifying the synth settings ring, allowing users to jump to the first or last setting, and to increase or decrease the current setting in a larger step.
-There are also new quick navigation commands, allowing userts to bind gestures to quickly jump between: paragraph, vertically aligned paragraph, same style text, different style text, menu item, toggle button, progress bar, figure, and math formula.
+There are also new quick navigation commands, allowing users to bind gestures to quickly jump between: paragraph, vertically aligned paragraph, same style text, different style text, menu item, toggle button, progress bar, figure, and math formula.
 
 There are many new braille features and bug fixes.
 A new braille mode called "display speech output" has been added.
-When active, the braille display shows exactly what NVDA prepares to speak.
+When active, the braille display shows exactly what NVDA speaks.
 Support was also added for the BrailleEdgeS2, BrailleEdgeS3 braille devices.

```

Comment:
```suggestion
Support was also added for the BrailleEdgeS2 and BrailleEdgeS3 braille devices.
```

Are these displays? If so, rather than redundantly using the word braille:
```suggestion
Support was also added for the BrailleEdgeS2 and BrailleEdgeS3 displays.
```